### ISSUE TITLE
fix broken link styling

### ISSUE THREAD

**Issue by abhijithvijayan** [ceadoor/cea.ac.in#56]
fix
https://ceadoor.github.io/cea.ac.in/studies/btech-computer-science-engg.html

**Comment by abhijithvijayan** [comment]
possible cause of webpack

**Comment by ashikmeerankutty** [comment]
@abhijithvijayan 
Try changing `publicPath: ""` to `publicPath: "/"` and then adding the path along with the filename in HtmlWebpackPlugin, 
```
new HtmlWebpackPlugin({
      template: "src/studies/btech-computer-science-engg.html",
      inject: "body",
      filename: "studies/btech-computer-science-engg.html"
    })
```

**Comment by abhijithvijayan** [comment]
@ashikmeerankutty it was the issue of the `main.js` not being served from `/`

**Comment by abhijithvijayan** [comment]
Closing.

**Comment by abhijithvijayan** [comment]
[Fix] #63

**Comment by ashikmeerankutty** [comment]
#64 fixed
